### PR TITLE
release v1.2.0

### PR DESCRIPTION
**Pull request by djrtwo**
DO NOT MERGE

### CODE REVIEW

**Review by hwwhww — APPROVED** [review]
note that there is a minor test format change (#3001)

testgen completed successfully here ✅

File: tests/formats/fork_choice/README.md [hunk 1]
```diff
@@ -106,8 +106,8 @@ Optional step for optimistic sync tests.
     block_hash: string,             -- Encoded 32-byte value of payload's block hash.
     payload_status: {
         status: string,             -- Enum, "VALID" | "INVALID" | "SYNCING" | "ACCEPTED" | "INVALID_BLOCK_HASH".
-        latestValidHash: string,    -- Encoded 32-byte value of the latest valid block hash, may be `null`.
-        validationError: string,    -- Message providing additional details on the validation error, may be `null`.
```
Comment (Joumax11) [inline tests/formats/fork_choice/README.md]: tests/formats/fork_choice/README.md

**Comment by Jiggnubs** [comment]
Am I getting close?

**Review by Influ20 — APPROVED** [review]
Yea

**Pull request merged**